app/src/main/java/com/mapbox/dash/example/MainActivity.kt

```diff
@@ -751,9 +725,7 @@ class MainActivity : DrawerActivity() {
 
         private const val TAG = "MainActivity"
 
-        private const val KEY_ORIGINAL_DENSITY = "original_density"
-        private const val KEY_ORIGINAL_SMALLEST_WIDTH = "original_smallest_width"
-        private const val KEY_TABLET_LAYOUT = "tablet_layout"
+        private var tabletLayout: Boolean? = null
```

Comment:
Can we init it using `lazy`?

Reply:
I don't think so, this is `companion object`

Reply:
Yeah, but it could probably be moved to Activity level.

Reply:
It must survive configuration changes. Moving to a view model or saving into a `Bundle` is not an option because then we won't be able to access the property in `attachBaseContext`